<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>AI4Science Lab</title>

    <link rel="stylesheet" type="text/css" href="http://127.0.0.1:4000//assets/css/styles_feeling_responsive.css">

  

	<script src="http://127.0.0.1:4000//assets/js/modernizr.min.js"></script>

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
			}
		});
	</script>

	<noscript>
		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
	<meta name="description" content="The AI4Science Lab is an initiative supported by the Faculty of Science (FNWI) at the University of Amsterdam and located in the Informatics Institute (IvI). The AI4Science Lab is also connected to AMLAB, the Amsterdam Machine Learning Lab.
We develop and use machine learning techniques to discover patterns in data streams produced by experiments in a wide variety of scientific fields, ranging from ecology to molecular biology and from chemistry to astrophysics.">
	<meta name="google-site-verification" content="Vk0IOJ2jwG_qEoG7fuEXYqv0m2rLa8P778Fi_GrsgEQ">
	<meta name="msvalidate.01" content="0FB4C028ABCF07C908C54386ABD2D97F" >
	
	<link rel="author" href="https://plus.google.com/u/0/118311555303973066167">
	
	
	<link rel="canonical" href="http://127.0.0.1:4000//research/">


	<!-- Facebook Open Graph -->
	<meta property="og:title" content="AI4Science Lab">
	<meta property="og:description" content="The AI4Science Lab is an initiative supported by the Faculty of Science (FNWI) at the University of Amsterdam and located in the Informatics Institute (IvI). The AI4Science Lab is also connected to AMLAB, the Amsterdam Machine Learning Lab.
We develop and use machine learning techniques to discover patterns in data streams produced by experiments in a wide variety of scientific fields, ranging from ecology to molecular biology and from chemistry to astrophysics.">
	<meta property="og:url" content="http://127.0.0.1:4000//research/">
	<meta property="og:locale" content="en_EN">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="AI4Science Lab">
	
	<meta property="article:author" content="https://www.facebook.com/phlow.media">


	
	<!-- Twitter -->
	<meta name="twitter:card" content="summary">
	<meta name="twitter:site" content="phlow">
	<meta name="twitter:creator" content="phlow">
	<meta name="twitter:title" content="AI4Science Lab">
	<meta name="twitter:description" content="The AI4Science Lab is an initiative supported by the Faculty of Science (FNWI) at the University of Amsterdam and located in the Informatics Institute (IvI). The AI4Science Lab is also connected to AMLAB, the Amsterdam Machine Learning Lab.
We develop and use machine learning techniques to discover patterns in data streams produced by experiments in a wide variety of scientific fields, ranging from ecology to molecular biology and from chemistry to astrophysics.">
	
	

	<link type="text/plain" rel="author" href="http://127.0.0.1:4000//humans.txt">

	

	

	<link rel="icon" sizes="32x32" href="http://127.0.0.1:4000/assets/img/favicon-32x32.png">

	<link rel="icon" sizes="192x192" href="http://127.0.0.1:4000/assets/img/touch-icon-192x192.png">

	<link rel="apple-touch-icon-precomposed" sizes="180x180" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-180x180-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-152x152-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-144x144-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="120x120" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-120x120-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-114x114-precomposed.png">

	
	<link rel="apple-touch-icon-precomposed" sizes="76x76" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-76x76-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-72x72-precomposed.png">

	<link rel="apple-touch-icon-precomposed" href="http://127.0.0.1:4000/assets/img/apple-touch-icon-precomposed.png">	

	<meta name="msapplication-TileImage" content="http://127.0.0.1:4000/assets/img/msapplication_tileimage.png">

	<meta name="msapplication-TileColor" content="#fabb00">


	

</head>
<body id="top-of-page" class="page-fullwidth">
	
	
<div id="navigation" class="sticky">
  <nav class="top-bar" role="navigation" data-topbar>
    <ul class="title-area">
      <li class="name">
      <h1 class="show-for-small-only"><a href="http://127.0.0.1:4000/" class="icon-tree"> AI4Science Lab</a></h1>
    </li>
       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->

<!-- ***** 25-1-2023: "menu-icon" removed by Bernd after failing to change its color to white in _sass/foundation-components/_top-bar.scss (look for "menu-icon" and "hamburger icon") *****  -->
      <!--  <li class="toggle-topbar menu-icon"><a href="#"><span>Nav</span></a></li>  -->
      <li class="toggle-topbar"><a href="#"><span>Menu</span></a></li>

    </ul>
    <section class="top-bar-section">

      <ul class="right">
        

              



          
          
        

              



          
          
        

              



          
          
            
            
              <li class="divider"></li>
              <li class="active"><a  href="http://127.0.0.1:4000//research/">Research</a></li>

            
            
          
        

              



          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://127.0.0.1:4000//education/">Education</a></li>

            
            
          
        

              



          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://127.0.0.1:4000//members/">Members</a></li>

            
            
          
        

              



          
          
            
            

              <li class="divider"></li>
              <li class="has-dropdown">
                <a  href="http://127.0.0.1:4000//events/">Events</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://127.0.0.1:4000//colloquium/">Colloquia</a></li>
                    

                      

                      <li><a  href="http://127.0.0.1:4000//events/">Workshops</a></li>
                    
                  </ul>

              </li>
            
          
        

              



          
          
            
            

              <li class="divider"></li>
              <li class="has-dropdown">
                <a  href="http://127.0.0.1:4000//publications/">Publications</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://127.0.0.1:4000//publications/">Papers</a></li>
                    
                  </ul>

              </li>
            
          
        

              



          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://127.0.0.1:4000//search/">Search</a></li>

            
            
          
        

              



          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://127.0.0.1:4000//contact/">Contact</a></li>

            
            
          
        
        
      </ul>

      <ul class="left">
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://127.0.0.1:4000//">AI4Science</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://127.0.0.1:4000//consortium/">Consortium</a></li>
                    

                      

                      <li><a  href="http://127.0.0.1:4000//lab/">Lab</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://127.0.0.1:4000//ai4smm_home/">AI4SMM</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://127.0.0.1:4000//ai4smm_rpa/">About</a></li>
                    

                      

                      <li><a  href="http://127.0.0.1:4000//ai4smm_people/">People</a></li>
                    

                      

                      <li><a  href="http://127.0.0.1:4000//ai4smm_students/">Bachelor / Master projects</a></li>
                    

                      

                      <li><a  href="http://127.0.0.1:4000//ai4smm_events/">AI4SMM Events</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        
        
      </ul>
    </section>
  </nav>
</div><!-- /#navigation -->

	

	




	


<div class="row t30">
	<div class="medium-12 columns">
		<article>
			<header>
				
				<h1></h1>
			</header>

			

			<center> <h1>Research</h1> </center>

<h2 id="gravitational-waves">Gravitational Waves</h2>
<p><br /></p>

<p><img src="../images/project_benjaminmiller1.png" alt="Project1" width="400" style="float: left; margin-right: 10px;" /></p>

<p>The breakthrough discovery of the first gravitational wave signal in September 2015 (Nobel Prize 2017) has opened a new window to the Universe. Analyses of the signal waveforms during the initial inspiral, merger and final ringdown phase provide crucial information about the properties of superheavy stellar objects.</p>

<p>By Benjamin Miller</p>

<p><img src="../members/BenjaminMiller.jpg" alt="Benjamin Miller" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />

<h2 id="liquid-chromatography">Liquid Chromatography</h2>
<p><br /></p>

<p><img src="../images/project_jimboelrijk1.png" alt="Project1" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Two-dimensional liquid chromatography (2DLC) is a powerful technique to separate and detect trace molecular compounds in complex samples (think of proteins, antibodies, oils, paints, etc.). The separation is based on the difference in “retention” time that it takes for each compound to be carried by a solvent (mobile phase) through a column filled with a material (stationary phase) that interacts with the injected compounds based on a chosen property, such as molecular size, charge or hydrophilicity, as schematically drawn on the right. 2DLC is combined with a detection method such as UV-VIS, IR, or mass spectroscopy (MS), that identifies the separated molecules. As the compounds can have extremely low concentrations (e.g. protein biomarkers, plant hormones, food contaminants), the signals of interest are often buried in the noise of the data and information is being lost. Typical datasets contain several gigabytes of data per measurement.</p>

<p><img src="../images/project_jimboelrijk1_thumb.jpg" alt="LC" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Analyzing and extracting all relevant information from such data is a challenging task. In addition, 2DLC method development is daunting as there are numerous parameters one can vary in order to achieve optimal separation. Successful implementation of the technique requires months of costly and cumbersome development. In response to this, algorithms are being developed to model chromatographic interaction of analyte molecules with the chemical moieties of the stationary phase, so as to allow prediction of optimal chromatographic conditions. In this project, we will look at the application of AI techniques, such as Bayesian inference, Bayesian optimization and deep learning to tackle parameter optimization, peak detection, baseline correction, and more.</p>

<p>By Jim Boelrijk</p>

<p><img src="../members/JimBoelrijk2.jpg" alt="Jim Boelrijk" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />

<h2 id="radio-astronomy">Radio Astronomy</h2>
<p><br /></p>

<p><img src="../images/fig_project3b.png" alt="LC" width="400" style="float: left; margin-right: 10px;" /></p>

<p>One of the cutting edge areas of astronomy is recognising rare features in data streams in near-real time. This problem has arisen from massive imaging surveys of the sky at many wavelengths (optical, radio, X-ray). The purpose is no longer just to find objects that are there all or most of the time, but to find so-called ‘transient’ objects that appear fleetingly in hindsight.</p>

<p>These transients are typically multi-messenger events, meaning that their emission is seen throughout the electromagnetic spectrum and using other ‘messengers’ including neutrinos and gravitational waves. By combining the multi-messenger observations, we can probe the most extreme physical environments in the universe. However, we are not monitoring the whole sky all the time with sensitive observatories, so we need to alert them to interesting objects in order to trigger follow-up observations. As these events are often fleeting at different wavelengths, we need to trigger the follow-up in as close to real-time as possible.</p>

<p><img src="../images/project_davidruhe1_thumb.png" alt="LC" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Due to the nature of our searches, we cannot always predict the transient behaviour we are searching for, nor all more rare specific instrumental faults in the data stream. The real-time data streams are subject to different types of behaviour, which we do not want to confuse with astrophysical transient events, including radio frequency interference, calibration artefacts, varying noise features, space weather, meteors, airplanes and satellites.</p>

<p>There are multiple areas where ML can assist with solving the task at hand and the challenges it comes with. Examples are: finding latent (disentangled) representations of observed phenomena, undoing data corruption (e.g. denoising), unsupervised outlier detection, geometric deep learning. The project will explore the application and innovation of these techniques and will push the boundaries of machine learning and astronomy.</p>

<p>By David Ruhe</p>

<p><img src="../members/DavidRuhe.jpg" alt="David Ruhe" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />

<h2 id="gene-regulatory-networks">Gene Regulatory Networks</h2>

<p><br /></p>

<p><img src="../images/project_teodorapandeva1.png" alt="Project1" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Causal discovery to reveal complex gene regulation networks from vast transcriptome datasets
Gene regulatory networks (GRNs) model the biological interactions between genes and provide a better understanding of the cellular processes and regulation pathways. Correctly constructed GRNs can play a fundamental role in solving various biological and biomedical problems, such as tracking disease development. Technological advances of high-throughput molecular biology have opened a new research frontier for inferring GRNs in a completely data-driven way. The great amount of gene expression data available on the GEO database suggests using AI-based techniques for processing and modeling such networks.
In this project, we focus on Bacillus subtilis, a well-studied Gram-positive bacterium with more than 3,000 publicly available transcriptome datasets. Combining these datasets alongside with reconstructing the known GRNs can be a very challenging task due to the (un)observable batch effects in the data that result from the varying data generating process across experiments. By tackling these issues and applying novel causal discovery techniques we aim for a better interpretation of the genes functions and their organization in hierarchical structures or even revealing unknown regulatory networks.</p>

<p>By Teodora Pandeva</p>

<p><img src="../members/TeodoraPandeva.jpg" alt="TeodoraPandeva" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />

<h2 id="radar-aeroecology">Radar Aeroecology</h2>

<p><br /></p>

<p><img src="../images/project_fionalippert1_thumb.png" alt="Source: doi:10.3390/rs11192233" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Every year millions of birds travel hundreds if not thousands of kilometers between their breeding and wintering grounds, seeking for seasonally optimal habitats. These mass movements are an essential component of biodiversity as they link otherwise separated ecosystems. Climate change and rapid expansions of human structures and activities may have severe implications for migrating birds. In order to assess the ecological consequences of these developments and to devise measures to preserve migration systems, it is essential to better understand continental-scale migration patterns and its external drivers such as weather, habitat quality and land use.</p>

<p>However, studying the aerial movement of migrating birds on a large scale is challenging. Many birds migrate during the night or at high altitudes which makes it impossible for humans to observe them from the ground. Modern tracking devices overcome this issue but they are limited to very few individuals and species with sufficient body mass. Radar technology is considered a promising alternative. In particular international networks of weather radars, which are known to capture mass movements of migrating birds, offer unparalleled opportunities to quantify and analyze bird migration at large spatial and temporal scales. But the field of radar aeroecology is still in its infancy. Challenges relating to the mere volume of raw radar observations as well as data quality issues such as inconsistent calibration and preprocessing, observation artifacts and partial observability prevent ecologists from utilizing the full potential of available radar databases.</p>

<p><img src="../images/project_fionalippert1.png" alt="Source: doi:10.3390/rs11192233" width="500" style="float: right; margin-right: 10px;" />
This project will explore the use of machine learning to overcome this hurdle. By applying and advancing techniques for system identification and predictive modeling of partially observed spatio-temporal dynamical systems we aim at enhancing both the understanding and predictability of continental-scale bird migration in Europe.</p>

<p>By Fiona Lippert</p>

<p><img src="../members/FionaLippert.jpg" alt="FionaLippert" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<p>Images from:</p>

<p><a href="https://www.mdpi.com/2072-4292/11/19/2233">Nilsson et al. (2018) Revealing patterns of nocturnal migration using the European weather radar network</a></p>

<p><a href="https://doi.org/10.1111/ecog.04003">Nussbaumer et al. (2019) A Geostatistical Approach to Estimate High Resolution Nocturnal Bird Migration Densities from a Weather Radar Network</a></p>

<hr />

<h2 id="protein-stabilization-for-vaccine-design">Protein Stabilization for Vaccine Design</h2>

<p><br /></p>

<p><img src="../images/project_congliu1.png" alt="VacinneDesign" width="400" style="float: right; margin-right: 10px;" /></p>

<p>Highly contagious viruses, such as SARS-CoV-2, can have a negative impact on the health of people and the stability of social resources. Vaccines are an effective means of stopping the widespread spread of viruses in the population by giving people immunity to a particular virus, thereby stopping them from spreading the virus and protecting those at risk of being infected. One of the core aspects of vaccine development is the development of antigens. One of the ways in which antigens are developed is by mutating one/some of the amino acids in the antigen, thus making the antigen stable, which in turn triggers an immune response and the production of a large number of antibodies.
Among the methods for finding a viable antigen, the structural based approach has worked well and is widely used. However, their success rate for single point variants is only 5% to 10%. As a result, it can take a considerable amount of time to design a stable antigen that is feasible with multiple variants. Thanks to the large amount of available protein structure/sequence data, deep learning-based approaches have recently achieved unprecedented success in research for proteins, e.g. protein property prediction and protein structure prediction. In this project, our goal is to apply the cutting-edge deep learning methods (e.g. Graph Neural Networks, Transformers, Equivariant Neural Networks, AlphaFold2 etc…) to model our problems. Allowing the model to suggest variant sites and variant amino acids for antigens with correspondingly predicted stabilities, leading to the help for designing of stable antigens.</p>

<p>By Cong Liu</p>

<p><img src="../members/CongLiu.jpg" alt="Cong Liu" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />

<h2 id="imaging-and-metrology-for-semiconductors">Imaging and Metrology for Semiconductors</h2>

<p><br /></p>

<p><img src="../images/fiber_metrology.jpg" alt="fibre_metrology" width="400" style="float: right; margin-right: 10px;" /></p>

<p>The ever-increasing computational power and ever-shrinking size of electronic devices have been enabled by rapid developments in optical lithography and the micro-electronics industry. State-of-the-art processors require reliable and fast nanoscale engineering methods with high production efficiency. The widespread access to computational power has already made a big impact on our daily lives. New future applications like VR, AR or AI technologies require even smaller and more powerful hardware. However, keeping up with the aggressive reduction of the semiconductor device feature sizes is a challenging task.</p>

<p>In this project, we will explore deep learning and Bayesian optimization techniques to push the boundaries of compact semiconductor metrology tools. Label-free optical imaging methods with a spatial resolution beyond the Abbe diffraction limit and a temporal resolution beyond the Nyquist limit are being developed to characterize multi-layer nanostructures. Several modern research areas such as computational imaging and compressive sensing are connected with cutting-edge Machine Learning tools to develop new methods of nanoscale imaging and metrology for the semiconductor industry. The project is running in collaboration with the Advanced Research Center for Nanolithography (ARCNL) and ASML, the world leader in the production of the lithography machines.</p>

<p>By Maximilian Lipp</p>

<p><img src="../members/MaximilianLipp.jpg" alt="MaximilianLipp" width="100" style="float: center; margin-right: 10px; border-radius:50%;" /></p>

<hr />


		</article>
	</div><!-- /.medium-12.columns -->
</div><!-- /.row -->




	
	    <div id="up-to-top" class="row">
      <div class="small-12 columns" style="text-align: right;">
        <a class="iconfont" href="#top-of-page">&#xf108;</a>
      </div><!-- /.small-12.columns -->
    </div><!-- /.row -->


    <footer id="footer-content" class="bg-grau">
      <div id="footer">
        <div class="row">
          <div class="medium-6 large-5 columns">
            <h5 class="shadow-black">About This Site</h5>

            <p class="shadow-black">
              The AI4Science Lab is an initiative supported by the Faculty of Science (FNWI) at the University of Amsterdam and located in the Informatics Institute (IvI). The AI4Science Lab is also connected to AMLAB, the Amsterdam Machine Learning Lab.
We develop and use machine learning techniques to discover patterns in data streams produced by experiments in a wide variety of scientific fields, ranging from ecology to molecular biology and from chemistry to astrophysics.
              <a href="http://127.0.0.1:4000///lab">More ›</a>
            </p>
          </div><!-- /.large-6.columns -->


          <div class="small-6 medium-3 large-3 large-offset-1 columns">
            
              
                <h5 class="shadow-black">UvA Websites</h5>
              
            
              
            
              
            
              
            
              
            
              
            

              <ul class="no-bullet shadow-black">
              
                
                  <li >
                    <a href="http://127.0.0.1:4000/"  title=""></a>
                  </li>
              
                
                  <li >
                    <a href="https://www.uva.nl" target="_blank"  title="UvA website">University of Amsterdam</a>
                  </li>
              
                
                  <li >
                    <a href="https://www.uva.nl/en/about-the-uva/organisation/faculties/faculty-of-science/faculty-of-science.html" target="_blank"  title="FNWI website">Faculty of Science</a>
                  </li>
              
                
                  <li >
                    <a href="https://ivi.uva.nl" target="_blank"  title="Informatic Institute">Informatics Institute (IvI)</a>
                  </li>
              
                
                  <li >
                    <a href="http://amlab.science.uva.nl/" target="_blank"  title="AMLAB">Amlab</a>
                  </li>
              
                
                  <li >
                    <a href="http://127.0.0.1:4000//"  title="Take me home country roads">AI4Science home</a>
                  </li>
              
              </ul>
          </div><!-- /.large-4.columns -->


          <div class="small-6 medium-3 large-3 columns">
            
              
                <h5 class="shadow-black">Contact Us</h5>
              
            
              
            
              
            
              
            
              
            

            <ul class="no-bullet shadow-black">
            
              
                <li >
                  <a href="http://127.0.0.1:4000/"  title=""></a>
                </li>
            
              
                <li >
                  <a href="http://127.0.0.1:4000/"  title="Visit us">Visiting Adress: <br/> Science Park 904 <br/> 1098 XH Amsterdam</a>
                </li>
            
              
                <li >
                  <a href="http://127.0.0.1:4000/"  title="">Postal Address: <br/> University of Amsterdam <br/> AI4Science Lab <br/> P.O. Box 94323 <br/> 1090 GH Amsterdam <br/> The Netherlands</a>
                </li>
            
              
                <li >
                  <a href="http://127.0.0.1:4000/"  title="">Tel: +31 (0)20 525 7519</a>
                </li>
            
              
                <li >
                  <a href="http://127.0.0.1:4000/"  title="">Email: ai4science.info@gmail.com</a>
                </li>
            
            </ul>
          </div><!-- /.large-3.columns -->
        </div><!-- /.row -->

      </div><!-- /#footer -->


      <div id="subfooter">
        <nav class="row">
          <section id="subfooter-left" class="small-12 medium-6 columns credits">
            <p> © 2020 UvA FNWI AI4Science Lab. All Rights Reserved |Design: <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>. |Header image: <a href="https://www.flickr.com/photos/mikemacmarketing/30212411048">Mike MacKenzie</a>.</p>
          </section>

          <section id="subfooter-right" class="small-12 medium-6 columns">
            <ul class="inline-list social-icons">
            
              <li><a href="http://github.com/ai4science-amsterdam" target="_blank" class="icon-github" title="Code repository"></a></li>
            
              <li><a href="https://twitter.com/ai4science_lab" target="_blank" class="icon-twitter" title="Follow us on Twitter"></a></li>
            
            </ul>
          </section>
        </nav>
      </div><!-- /#subfooter -->
    </footer>

	

	


<script src="http://127.0.0.1:4000//assets/js/javascript.min.js"></script>














</body>
</html>

