---
layout: page-fullwidth 
show_meta: false
title: "AI4Science Colloquium"
#subheadline: "Welcome to our lab."
#teaser: "Hi, welcome to our lab."
header:
   image_fullwidth: "banner.jpg"
   title: ''
   caption: "Credits: Mike Mackenzie"
   caption_url: https://www.flickr.com/photos/mikemacmarketing/30212411048
permalink: "/colloquium/"
---
> Knowledge Shared = Knowledge<sup>2</sup>


The AI4Science Colloquium is a bi-weekly colloquium series, held on alternating Tuesdays at 14:00 Central European Time. In this colloquium our very own Teodora Pandeva and Fiona Lippert invite renowned speakers to present and discuss their state-of-the-art AI solutions for scientific discovery. Interested? Subscribe to our Email-list to be notified.

## Email List
To stay up to date with our activities and be invited to our biweekly AI4Science colloquium series, you may send a request to be included in our emaillist via [an email to us][9] with your name, affiliation and a one-sentence motivation for joining.


## Next Colloquium

#### Title: Representation and regression problems for molecular structure and dynamics 

Date: 27-10-2020 14:15-15:15 Central European Summer Time

 <img src="../people/GaborCsanyi.png"
     alt="gaborcsanyi"
     width="100"
     style="float: right; margin-right: 10px; border-radius:50%;" />

Speaker: **Gábor Csányi**, Professor of Molecular Modelling, Engineering Laboratory, University of Cambridge

**Abstract:** <br/>
A vast proportion of total global “science” computer time is spent on property prediction  and optimisation on the atomic scale in the fields of materials, physics and chemistry. Typical tasks include solving the Schrödinger equation in various approximations, molecular dynamics using Newtonian equations of motion, but also classification and design based on predicted interactions of molecules and periodic crystal structures. Efficient representations of material and molecular structure, coupled with regularisation (both in kernel methods and neural networks) are beginning to show dramatic speedups and also reductions in the scaling of computational complexity at the same time as making step-changes in simulation accuracy. The first science applications that take advantage of the million-factor speedups include accurate calculations of the structure of amorphous materials and molecular liquids. Outstanding problems include, but with good prospects, finding better low dimensional representations (e.g. body-ordered expansions), incorporating infinite range interactions, clarifying the relationships between low dimensionality and interaction range, and providing global error bounds. One of the most pressing problems is “extrapolation”: typical simulations generate samples using stochastic processes based on the fitted models and thus underestimating the energy can lead to exponential amplification of errors.

<!--
<img src="../images/physml.jpg"
     alt="Physical Machine learning"
     width="400"
     style="float: center; margin-right: 10px;"/>
<p>Ref: <a href="https://aip.scitation.org/doi/10.1063/1.5009502">Journal of Chemical Physics</a></p>
-->

## Schedule
* 27 October 2020 - **Gábor Csányi**
* 24 November 2020 - **Luisa Lucie-Smith**

## Previous Colloquia

#### Title: Attributing variance in single-cell genomics

Date: 13-10-2020 14:00-15:00 Central European Summer Time

 <img src="../people/DavidFischer.jpg"
     alt="davidfisher"
     width="100"
     style="float: right; margin-right: 10px; border-radius:50%;" />

Speaker: **David Fischer**, Theis Lab, ICB, Helmholtz Zentrum, Munich, Germany

**Abstract:** <br/>
Cell Biology revolves around the idea that cells are core functional units of organisms: Models for cellular function explain many disease and treatment effects, for example. Hence, there has been a long-standing, significant interest in improving the understanding of cellular behaviour. Recent experimental advances enable increasingly detailed molecular profiling of cells: Examples include methods such as single-cell RNA sequencing, which measure quantities of particular molecular species in each cell. The resulting output can be summarised as a matrix (cells x molecular features) with up to millions of cells and 10,000s of features. Models for variance in these data sets are actively developed in Computational Biology; many concepts from unsupervised learning are used in this context. While these unsupervised techniques yield useful descriptions of the molecular state space, they usually fail to yield a complete picture of molecular mechanisms of causality underlying this variance. Here, I discuss approaches centred on supervision tasks in single-cell Immunology which improve interpretability of these models and outline ways of generalising this approach to other settings in single-cell Biology. Lastly, I discuss challenges and opportunities of representation learning on whole tissues, bridging the gap from cell-centric models to tissue function models, such as for tumor behaviour.

<!--
<img src="../images/physml.jpg"
     alt="Physical Machine learning"
     width="400"
     style="float: center; margin-right: 10px;"/>
<p>Ref: <a href="https://aip.scitation.org/doi/10.1063/1.5009502">Journal of Chemical Physics</a></p>
-->
 
<a class="radius button small" href="https://drive.google.com/file/d/1uFoAVUAY3ges4HVneIe-V78IR8NoXIkp/view?usp=sharing">Watch Back ›</a>

For an overview of more  previous colloquia, please have a look at out [blog][2].

[1]: https://bereau.group/
[2]: /blog/
[9]: /contact/
